pkgpanda/constants.py

```diff
@@ -1,15 +1,28 @@
+from pkgpanda.util import is_windows
+
 RESERVED_UNIT_NAMES = [
     "dcos.target",
     "dcos-download.service",
     "dcos-setup.service"
 ]
 
+if is_windows:
+    STATE_DIR_ROOT = 'c:/var/lib/dcos'
+    PACKAGES_DIR = "packages"
+    PKG_DIR = "c:/pkg"
+    DOCKERFILE_DIR = 'docker.windows/'
+    config_dir = 'c:/etc/mesosphere'
+    install_root = 'c:/opt/mesosphere'
+    repository_base = 'c:/opt/mesosphere/' + PACKAGES_DIR
+else:
+    STATE_DIR_ROOT = '/var/lib/dcos'
+    PACKAGES_DIR = "packages"
+    PKG_DIR = "/pkg"
+    DOCKERFILE_DIR = 'docker/'
+    config_dir = '/etc/mesosphere'
+    install_root = '/opt/mesosphere'
+    repository_base = '/opt/mesosphere/' + PACKAGES_DIR
```

Comment:
When can accomplish this without duplicating these paths.  Something like this:

```
abs_path_prefix = '/'
if is_windows:
    abs_path_prefix = 'c:/'

STATE_DIR_ROOT = abs_path_prefix + 'var/lib/dcos'
# etc
```

Reply:
We will probably be setting this to c:/mesosphere/ so we can keep all these paths together. On windows we do not have /etc/, /opt, /var, so grouping everything under one directory would be more windows/. Then install_root may be the same as the prefix.